configure.ac

```diff
@@ -1585,7 +1585,11 @@ AM_CONDITIONAL([ENABLE_ZMQ], [test "x$use_zmq" = "xyes"])
 
 AC_MSG_CHECKING([whether to build test_bitcoin])
 if test x$use_tests = xyes; then
-  AC_MSG_RESULT([yes])
+  if test "x$enable_fuzz" = "xyes"; then
+    AC_MSG_RESULT([no, becasue fuzzing is enabled])
```

Comment:
I read "barbecue" here 🥩

Reply:
lol! I checked the output with all four test/fuzz combinations and failed to see the typo every time.